### PR TITLE
clippy and formatting

### PR DESCRIPTION
**Pull request by remkop22**

### CODE REVIEW

File: ocpi-tariffs/src/restriction.rs [hunk 1]
```diff
@@ -168,7 +169,7 @@ impl Restriction {
                 .map_or(true, |current| current < max_current),
             Self::MinPower(min_power) => state.min_power.map_or(true, |power| power >= min_power),
             Self::MaxPower(max_power) => state.max_power.map_or(true, |power| power < max_power),
-            Self::Reservation => todo!(),
+            // TODO: implement reservation restrictions (RP).
```
Comment (bheylin) [inline ocpi-tariffs/src/restriction.rs:172]: Remove the todo from the source and add it as a Github issue

File: ocpi-tariffs/src/types/number.rs [hunk 2]
```diff
@@ -84,7 +84,7 @@ impl Sub for Number {
     type Output = Self;
 
     fn sub(self, rhs: Self) -> Self::Output {
-        Self(self.0 / rhs.0)
+        Self(self.0 - rhs.0)
```
Comment (bheylin) [inline ocpi-tariffs/src/types/number.rs:87]: I would consider using `saturating_sub` and the other safe variants for all arithmetic operations.
At the moment you wrapping will occur silently.

**Review by bheylin — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -2,8 +2,9 @@
 members = ["cli", "ocpi-tariffs"]
 
 [workspace.dependencies]
-ocpi-tariffs = { path = "ocpi-tariffs" }
-cli = { path = "cli", package = "ocpi-tariffs-cli" }
+
```
Comment (bheylin) [inline Cargo.toml:5]: What's with the whitespace?

File: ocpi-tariffs/src/types/money.rs [hunk 4]
```diff
@@ -73,15 +73,15 @@ impl Add for Money {
     type Output = Self;
 
     fn add(self, rhs: Self) -> Self::Output {
-        Self((self.0 + rhs.0).into())
+        Self(self.0 + rhs.0)
```
Comment (bheylin) [inline ocpi-tariffs/src/types/money.rs:76]: You could also have saturating ops here

**Review by bheylin — CHANGES_REQUESTED** [review]

File: ocpi-tariffs/src/types/money.rs (hunk 4 above)
Comment (remkop22) [reply ocpi-tariffs/src/types/money.rs:76]: This uses the the `Add` impl of OCPI specific `Number` that already uses saturating. As does almost every other numeric type in the crate.

**Review by bheylin — APPROVED** [review]

**Pull request merged**